files/scripts/asic_status.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+function check_asic_status() {
+    chassis_sup=0
+    if [ -f /etc/sonic/chassisdb.conf ]; then
+        chassis_sup=1
+    fi
+
+    #Ignore
```

Comment:
Can we make a check here to exit out if we are not a chassis/supervisor ? We need not do this check for fixed platforms,

Reply:
Fixed. Please review.